### ISSUE TITLE
Make gif autoplay off by default

### ISSUE THREAD

**Issue by clarcharr** [tootsuite/mastodon#3876]
From an accessibility standpoint, it's a bigger issue if gifs autoplay than if you have to click a button to play them. This should be off by default.

A bit of clarification: the worst that can happen if the autoplay is off by default is the user can push a button. The worst that can happen if the autoplay is on by default is someone could have a seizure. This is a bit of an extreme case, but it's still a very good reason to change this feature in favour of accessibility.

Additionally, I think that it'd be very worthwhile to turn this feature off for all users once Mastodon is updated, allowing people to opt-in if they want. Most of the current users have not touched this setting, and it seems fair to revert in favour of accessibility while still making turning it back on as easy as a few clicks.

* * * *

- [X] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [X] This bug happens on a [tagged release](https://github.com/tootsuite/mastodon/releases) and not on `master` (If you're a user, don't worry about this).

**Comment by Dostoi** [comment]
There's an option in preferences that allow to disable the autoplay

**Comment by clarcharr** [comment]
I know. This is asking to make that the default.

**Comment by Dostoi** [comment]
Oh ok my bad. I though it was already

**Comment by ggtea** [comment]
Disable auto playing by default seems good for Mobile environment which has traffic limit.
I think disable auto playing isn't required for PC with FTTH which allows unlimited traffic.

**Comment by clarcharr** [comment]
@ggtea it isn't about traffic; it's more that gifs simply should not autoplay from an accessibility standpoint. Flashing images can trigger seizures for some and although most gifs are not too flashy, it's better to be safe. We don't have advertising revenue to gain by forcing people to watch things, so, I'd rather this be turned off by default.

**Comment by Cassolotl** [comment]
>We don't have advertising revenue to gain by forcing people to watch things, so, I'd rather this be turned off by default.

A very important point!

**Comment by Kimik0** [comment]
Even if it's not flashy/seizure-inducing, it's still distracting when you're trying to read other things than the image.

**Comment by staticsafe** [comment]
I will re-iterate my comment on issue #4122 : 

"Default the un-authed view to not have GIFs auto-play, have it click-to-play/hover-to-play. Auto-playing media is not only distracting but actively harmful for folks with something like photo-sensitive epilepsy.

What I am saying is that, pick a better default please."

For consistency's sake the default should be no auto-play in both authed and unauthed contexts.

**Comment by mostlyghost** [comment]
Yes please. 

My comment from #4122: 
gif autoplay? why isn't this opt-in, in order to make instances more accessible to people who need to avoid certain animations for health reasons? I really wish accessibility would be made more of a priority. :(

the response I received was that accessibility feature requests are a "niche issue." this is absolutely unacceptable. this is putting people in danger by potentially triggering seizures and other health problems which can be deadly. (no, I'm not exaggerating.) there's already been a fair number of instances that utilize a lot of animated gifs, and people can opt-into those by following users on those instances and joining those instances. but having a new user thinking "hmm, i wonder what this instance is like," while loading up the public timeline, and is then assaulted by flashing and strobing animation because auto-playing animation is "the expected behaviour"... well, honestly? that's ableist violence. especially because this auto-play gif issue has been talked about for months. this is not a new debate. 

i am asking as kindly and respectfully as i can: please reconsider a11y's priority within the mastodon project.

**Comment by Cassolotl** [comment]
I definitely agree that considering accessibility a "niche issue" is not acceptable. I would much prefer gif autoplay to be off. If you're concerned that will annoy abled people, perhaps make the setting easier to find and change?

**Comment by Gargron** [comment]
>niche issue

I didn't express myself clearly. The original comment said that Mastodon was not different to other social networks because of this issue. I said that there are plenty of things that make Mastodon different, and this one issue does not erase them. Using the word niche was an unlucky choice, because I was referring to its size relative to all the other things we're doing right.

**Comment by Gargron** [comment]
Resolved with #4132
